internal/goofys.go

```diff
@@ -1046,8 +1046,8 @@ func (fs *Goofys) MkDir(
 	op *fuseops.MkDirOp) (err error) {
 
 	fs.mu.RLock()
+	defer fs.mu.RUnlock()
```

Comment:
Need to unlock after MkDir otherwise, the MkDir creates duplicate inode for same name when MkDir is called with same name.